projects/bearn/index.js

```diff
@@ -0,0 +1,50 @@
+const { sumTokens2 } = require("../helper/unwrapLPs");
+
+const vaultFactory = "0x70b14cd0Cf7BD442DABEf5Cb0247aA478B82fcbb";
+const voter = "0x37d1E4594ed04818B68aF5396bc7388c26F17E4A";
+const bgt = "0x656b95E550C07a9ffe548bd4085c72418Ceb1dba";
+
+async function vaultsTVL(api) {
+  // grab all the vaults
+  const compoundingVaults = await api.call({
+    abi: "function getAllCompoundingVaults() external view returns (address[])",
+    target: vaultFactory,
+  });
+  const bgtEarnerVaults = await api.call({
+    abi: "function getAllBgtEarnerVaults() external view returns (address[])",
+    target: vaultFactory,
+  });
+
+  const vaults = [...compoundingVaults, ...bgtEarnerVaults];
+
+  // grab all the tokens and balances in the vaults
+  const tokens = await api.multiCall({
+    abi: "address:stakingAsset",
+    calls: vaults,
+  });
+
+  const balances = await api.multiCall({
+    abi: "uint256:totalAssets",
```

Comment:
can we use erc20:balanceOf here?

Reply:
the assets are staked in Berachain's reward vaults so we can't use stakingAsset.balanceOf() here

we can fetch Berachain's reward vaults with another multicall and use the balanceOf() for those instead if that's preferred

![image](https://github.com/user-attachments/assets/2e834129-2bbd-4684-b689-4ca20e8a1016)